recipes-games/pinball/pinball_0.3.20201218.bb

```diff
@@ -0,0 +1,77 @@
+SECTION = "games"
+LICENSE = "GPLv2 & CC0-1.0"
+LIC_FILES_CHKSUM = "file://COPYING;md5=94d55d512a9ba36caa9b7df079bae19f"
+LIC_FILES_CHKSUM += "file://debian/copyright;md5=ef0d1a481815636a22c1d73b1233d9c0"
```

Comment:
A single term with = would be easier to read

Reply:
OK